Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:carbon-alpine
+
+WORKDIR /usr/src/app
+
+COPY . /usr/src/app/
+RUN yarn install
```

Comment:
I would suggest passing in the `--frozen-lockfile` option here so that the build fails hard in case an update of the `yarn.lock` is necessary.